Tests/src/PublicAPITests/QuotaSystemTests.cpp

```diff
@@ -436,4 +446,25 @@ CSP_PUBLIC_TEST(CSPEngine, QuotaSystemTests, GetTotalSpaceSizeinKilobytes)
 	// Delete space
 	DeleteSpace(SpaceSystem, Space.Id);
 }
+#endif
+
+#if RUN_ALL_UNIT_TESTS || RUN_QUOTASYSTEM_TESTS || RUN_QUOTASYSTEM_STRINGTOEQUOTERESULTFAILUREREASONINTVALUE_TEST
+CSP_PUBLIC_TEST(CSPEngine, QuotaSystemTests, StringToEQuotaResultFailureReasonIntValueTest)
+{
+	EXPECT_EQ(StringToEQuotaResultFailureReasonIntValue(""), static_cast<int>(EQuotaResultFailureReason::None));
```

Comment:
This should be equal to `Unknown` and not `None`. `None` is _only_ used when there is no error.